src/test/java/de/uhd/ifi/se/decision/management/jira/persistence/codeclassknowledgeelementpersistencemanager/TestGetDecisionKnowledgeElement.java

```diff
@@ -0,0 +1,82 @@
+package de.uhd.ifi.se.decision.management.jira.persistence.codeclassknowledgeelementpersistencemanager;
+
+import com.atlassian.jira.user.ApplicationUser;
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.impl.KnowledgeElementImpl;
+import de.uhd.ifi.se.decision.management.jira.persistence.CodeClassKnowledgeElementPersistenceManager;
+import de.uhd.ifi.se.decision.management.jira.testdata.JiraUsers;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class TestGetDecisionKnowledgeElement extends TestSetUp {
+
+	private KnowledgeElement classElement;
+	private KnowledgeElement classElement2;
+	private CodeClassKnowledgeElementPersistenceManager ccManager;
+	private ApplicationUser user;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'user' could be replaced by a local variable.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5229259)